src/main/resources/application.yaml

```diff
@@ -17,7 +17,7 @@ spring:
 
 queue:
   connection-string: ${QUEUE_CONNECTION_STRING:}
-  read-interval: ${QUEUE_READ_INTERVAL:30000} # in ms
+  read-interval: ${QUEUE_READ_INTERVAL:500} # in ms
```

Comment:
is this supposed to be here?

Reply:
ahh .. a testing comit I should put it in my application default. Mistakenly committed